train.py

```diff
@@ -969,6 +969,19 @@ def main():
                         "add_v_proj",
                         "to_add_out.0",
                     ]
+                elif args.flux_lora_target == "full":
```

Comment:
```suggestion
                elif args.flux_lora_target == "all+norms":
```

Reply:
It's not norms, it's ff. I'll change it to "all+ffs"